geomstats/_backend/tensorflow/random.py

```diff
@@ -42,3 +42,7 @@ def uniform(low=0.0, high=1.0, size=None):
     if size is None:
         size = (1,)
     return tf.random.uniform(shape=size, minval=low, maxval=high)
+
+
+def multivariate_normal(*args, **kwargs):
```

Comment:
How about this:
https://www.tensorflow.org/probability/api_docs/python/tfp/distributions/MultivariateNormalFullCovariance

Reply:
Yes, but we have to add the dependence on the tensorflow probability package, it is a separate one